zebra-state/src/service/check/utxo.rs

```diff
@@ -200,7 +200,7 @@ pub fn transparent_coinbase_spend(
     match spend_restriction {
         OnlyShieldedOutputs { spend_height } => {
             let min_spend_height =
-                utxo.utxo.height + block::Height(MIN_TRANSPARENT_COINBASE_MATURITY);
+                utxo.utxo.height + MIN_TRANSPARENT_COINBASE_MATURITY.try_into().unwrap();
```

Comment:
Optional: It would be nice for `MIN_TRANSPARENT_COINBASE_MATURITY` to be a `HeightDiff` too.

Reply:
I think this might cause issues with the type in other code, so that seems like something for a later cleanup after the audit.

Reply:
I added a TODO for this.